docs/design/disqualification.md

```diff
@@ -0,0 +1,83 @@
+
+> When a node operator wants to leave the network, if they just shut down the node, the unavailability of the data on the network can contribute to repair costs.  If instead, the node triggers a function to call a Satellite, request new storage nodes to store the pieces stored on the node being shut down, then directly upload those pieces to the new nodes, file repair would be avoided.
+
+## Design
+
+### Use of Disqualified Nodes
+
+Disqualified nodes may be used during download of typical downloads from uplinks or via repair.  Therefore, their IP must be tracked in the overlay cache.
+
+Disqualified nodes may not be used for upload, therefore they should not be returned from node selection proceses of any sort.  There is no reason to update the statistics of disqualified nodes.
+
+The list of disqualified nodes should change infrequently, but could grow large over time.  If in the long run, the list of disqualified nodes becomes very large, it may benefit us to move it to its own data store rather than using a disqualified flag.
+
+### Handling Disqualified Nodes
+
+Disqualification can be handled in our existing SQL implementation by adding a `disqualified` column to the `nodes` table:
+
+```sql
+CREATE TABLE nodes (
+  id bytea NOT NULL,
+  ...
+  disqualified boolean NOT NULL,
+  PRIMARY KEY ( id )
+);
+```
+
+Existing SQL queries employing logic such as `WHERE audit_success_ratio >= $2 AND uptime_ratio >= $3` would change to `WHERE disqualified = FALSE`.
+
+Existing calls to the DBX `UpdateNodeInfo()` method must set `disqualified` if appropriate.  Care should be employed to not overburden the data structures used to store node info.  In the case of Postgres, these tables may be updated and return their values in a single SQL statement.
+
+### Determining Disqualification
+
+A node is disqualified when its reputation falls below a fixed value.  We are currently envision two disinct reputation check values, one for uptime and another for audit.  These values will represent an idea value minus some some standard-deviation.  The proposed system for calculation repuations is based on four values: α0, β0, λ, and v.  Because changing these values will change the expected standard deviation of measurements, the reputation cutoff values will vary as these parameters vary.  At this phase, it is exected that these cutoffs are all configured based on numbers from the data science team.  A node will be disqualified if either the audit or the uptime reputation value falls below their disqualification cutoff value.
+
+## Rationale
+
+Although disqualification is largely an atomic operation that would be handled well by an external hash, the inherent tie-ins with node selection make the above solution the most straightforward.  If we were to refactor node selection in the future, I would likely leave disqualified nodes out of the stats database, leaving them only in the overlay cache.
```

Comment:
"I would likely leave disqualified nodes" you may want to write "We would likely..."